src/hivpy/cli.py

```diff
@@ -18,21 +19,24 @@
 def run_model():
     '''
     Running a HIV model simulation
-    Assuming there is a hivpy.conf file the command is
-    $run_model hivpy.conf
+    Assuming there is a hivpy.yaml file the command is
+    $run_model hivpy.yaml
     '''
     parser = argparse.ArgumentParser(description="run a simulation")
-    parser.add_argument("input", type=pathlib.Path, help="run_model config.conf")
+    parser.add_argument("input", type=pathlib.Path, help="run_model config.yaml")
     args = parser.parse_args()
-    conf_filename = args.input
-    config = configparser.ConfigParser()
+    config_filename = args.input
+    try:
+        with open(config_filename, 'r') as file:
+            config = yaml.safe_load(file)
+    except yaml.YAMLError as err:
+        print("Error parsing yaml file {}".format(err))
```

Comment:
Maybe we can print the error in a more readable way, but probably fine to leave this until we get actual errors and feedback from users.